train.py

```diff
@@ -120,12 +120,18 @@ def weight_stats(self, iteration, model, log_weight_distribution, eps = 1e-9):
 				self.summary_writer.add_histogram(tag + '/grad', param.grad, iteration)
 	
 
-def apply_model(data_loader, model, generator, text_pipelines, device, oom_handler):
+def apply_model(data_loader, model, generator, text_pipelines, device, oom_handler, onnx_wrapper=None, forward_x_only=False):
```

Comment:
the final metrics seem to match, so no need to pass through onnx_wrapper, you can just assign `model` to `OnnxWrapper` instance

Reply:
done